packages/components/pill/src/Pill.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { CommonProps, Box } from '@contentful/f36-core';
+import { TabFocusTrap } from '@contentful/f36-utils';
+import { Drag, Close } from '@contentful/f36-icons';
+import { PillVariants } from './types';
+import getPillStyles from './styles';
+
+export interface PillProps
+  extends React.HTMLAttributes<HTMLElement>,
```

Comment:
Do we actually want to allow all `React.HTMLAttributes<HTMLElement>` to be passed or just specific props + CommonProps will be enough?

Reply:
I started wondering because the props table generated automatically looks huge

Reply:
That's a good question, I kept it with the `React.HTMLAttributes<HTMLElement>` as it was like that on the v3, but we can remove it if it's not necessary.
Or maybe only use the `React.HTMLAttributes<HTMLDivElement>` it should remove some of the props from the generated table.
WDYT??